lms/static/sass/_header.scss

```diff
@@ -0,0 +1,344 @@
+            }
+          }
+        }
+      }
+
+      .secondary {
+        @include float(right);
+
+        margin: $baseline*0.6 $baseline 0 0;
+
+        // All navigation items
+        .nav-item {
+          font-size: $font-size-base;
+          display: inline-block;
+          padding: $baseline/2 0 0;
+          margin: 0 $baseline/2;
+
+          a {
+            color: theme-color("dark");
+            font-weight: $font-weight-bold;
+          }
+        }
+
+
+        // Sign in, Register and Shopping Cart buttons
+        .btn {
+          margin: 0 $baseline/4 0;
+          border: 1px solid theme-color("primary");
+          padding: $baseline/4 $baseline;
+          border-radius: $baseline/4;
+          cursor: pointer;
+          text-decoration: none;
+        }
+
+        .sign-in-btn,
+        .nav-item a.shopping-cart {
+          background-color: theme-color("primary");
+          color: theme-color("inverse");
+          border: 1px solid theme-color("inverse");
+          font-weight: $font-weight-normal;
+          padding: $baseline/4 $baseline/2;
+
+          &:hover {
+            background-color: theme-color("inverse");
+            color: theme-color("primary");
+            border-color: theme-color("primary");
+          }
+        }
+
+        .register-btn {
+          background: theme-color("inverse");
+          color: theme-color("primary");
+
+          &:hover {
+            background-color: theme-color("primary");
+            color: theme-color("inverse");
+          }
+        }
+
+        // User information
+        .user-image-frame {
+          border: 1px solid theme-color("light");
+          margin: -1*$baseline/2 $baseline/4;
+          border-radius: $baseline/4;
+          width: $header-user-image-size;
+        }
+
+        // Dropdown behavior
+        .toggle-user-dropdown {
+          text-decoration: none;
+          cursor: pointer;
+        }
+
+        .dropdown-user-menu {
+          display: none;
+          border: 1px solid theme-color("secondary");
+          position: absolute;
+          background-color: theme-color("inverse");
+          color: theme-color("secondary");
+          right: 30px;
+          top: 50px;
+          z-index: 10;
+
+          .dropdown-item {
+            padding: $baseline*0.4 $baseline;
+            cursor: pointer;
+            font-weight: $font-weight-bold;
+
+            &:hover {
+              color: theme-color("dark");
+              background-color: theme-color("light");
+            }
+
+            a:focus {
+              outline: none;
+              color: theme-color("primary");
+            }
+
+            &:not(:last-child) {
+              border-bottom: 1px solid theme-color("light");
+            }
+          }
+        }
+      }
+    }
+
+    .hamburger-menu {
+      display: none;
+    }
+  }
+
+  // Responsive styling for mobile
+  @include media-breakpoint-down(sm) {
```

Comment:
As earlier, I'd like to see this implemented as mobile first with the additional behavior added with `media-breakpoint-up`.

Reply:
For this one, since it has so much code and it is only relevant for mobile, I'm going to leave as is - it would require adding a lot of overrides that makes the code more complex.